Watch Extension/AddFeedingView.swift

```diff
@@ -13,36 +13,35 @@ struct FeedingOptionsView: View {
     @Binding var feedingIntent: FeedingType
     let activeFeedingTypes: [FeedingType]
 
-    private func opacity(for type: FeedingType) -> Double {
-        isAdding(feeding: feedingIntent) || activeFeedingTypes.contains(type)
-            ? 0.5 : 1.0
-    }
-
     var body: some View {
         VStack {
             Spacer()
             HStack {
                 Spacer()
-                Image(systemName: "n.circle.fill")
-                    .font(.system(size: 55))
-                    .scaleEffect(isAdding(feeding: feedingIntent) ? 0.5 : 1)
-                    .blur(radius: isAdding(feeding: feedingIntent) ? 1.0 : 0)
-                    .opacity(opacity(for: .nursing))
-                    .disabled(activeFeedingTypes.contains(.nursing))
-                    .gesture(TapGesture().onEnded {
-                        withAnimation(.spring(response: 0.35, dampingFraction: 0.65)) { self.feedingIntent = .nursing }
-                    })
+                Button(action: {
+                    withAnimation(.spring(response: 0.35, dampingFraction: 0.65)) { self.feedingIntent = .nursing }
+                }) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/00b075433a67ac49d28f4d4df65587fdb4213111/Watch Extension/AddFeedingView.swift#L23"></span>Trailing closure syntax should not be used when passing more than one closure argument.<br />
<code>multiple_closures_with_trailing_closure</code> <code>AddFeedingView.swift:23</code></td>
    </tr>
  </tbody>
</table>